spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/ChannelBindingServiceConfiguration.java

```diff
@@ -83,6 +85,9 @@
 	@Autowired
 	private MessageBuilderFactory messageBuilderFactory;
 
+	@Lazy @Autowired(required = false)
```

Comment:
This is because of the `CompositeMessageConverterFactory` which is lazily initialised as well as autowired `ObjectMapper`.